### DISCUSSION TITLE
Thoughts on adding Collaboration section on the top navigation.

### DISCUSSION THREAD

**Discussion by fanalis93** [community/community#78011] [Other Feature Feedback, Questions, & Ideas]
**JUST A QUICK THOUGHT**
As a developer collaborating in multiple repos, it's a hassle for me to go through Settings->Repositories->Scroll all the way down, to just access the repos that I am collaborating. I think it would be great just add that Collaboration section on the top nav access on one-click. 
![image](https://github.com/community/community/assets/28630547/3ea9d606-b024-4f29-a493-3b8b2c4704f6)

**Comment by mdemonahmed** [comment]
yeah. I think it will be best if add another tab called "Collaboration" . under this show all repo lists that collaborate with the user. with time and repo title. 
it will be easy for all users to find easily.

**Comment by cehfisher** [comment]
Thanks for posting in the GitHub Community, @fanalis93! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've asked @queenofcorgis to move this issue into the correct category for you. Good luck!
